File: languages/java/testData/variable/variableDeclaration.java
package testData.variable;

public class MyClass {
    public int myFi<caret>eld = 0;
    public static int colli<caret>sion = 1;

    public void setField(int b<caret>oi) {
        myField = boi;
    }

    public static void main(String[] arg<caret>s) {
        MyClass insta<caret>nce = new MyClass();
        instance.myField = 1;
    }
}

File: languages/kotlin/testData/variable/variableDeclaration.kt
package variable

class MyClass4(not_colli<caret>sion: Boolean, val collis<caret>ion1: Int = 0) {
    var myFie<caret>ld = 0
    private val collis<caret>ion2 = 1
    fun setField(bo<caret>i: Int) {
        myField = boi
    }

    companion object {
        @JvmStatic
        fun main(ar<caret>gs: Array<String>) {
            val inst<caret>ance = MyClass4(true)
            instance.myField = 1
        }
    }
}